addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -530,7 +530,12 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
 
             var tour = self.tours[tour_name];
             if (typeof tip.run === "function") {
-                tip.run.call(tip.widget, action_helper);
+                try {
```

Comment:
This one could have been in a separate commit, as we might be considering backporting that part?